blitz/blitz.h

```diff
@@ -86,13 +86,23 @@ namespace blitz {
  using namespace std;
 #endif
 
+#ifdef WIN32
+#ifdef BZ_GENERATE_GLOBAL_INSTANCES
+#define _bz_global __declspec(dllexport)
+#define BZ_GLOBAL_INIT(X)   =X
+#else
+#define _bz_global __declspec(dllimport)
+#define BZ_GLOBAL_INIT(X) 
+#endif
+#else
 #ifdef BZ_GENERATE_GLOBAL_INSTANCES
  #define _bz_global
  #define BZ_GLOBAL_INIT(X)   =X
 #else
  #define _bz_global extern
  #define BZ_GLOBAL_INIT(X) 
 #endif
+#endif
 
 /* Define types for indexing, depending on whether 64- or 32-bit
    indices are desired. There are separate typedefs for sizeType and
```

Comment:
You may be thinking of GenerateExportHeader. To use that it will require a bunch of changes. I did update the file from #ifdef WIN32 to #ifdef _WIN32 to match the check in the BZ_THREADSAFE block. (#elif defined(_WIN32)), so the check is the same in both spots.

I did not see a way to this included in the bzconfig.h file without the #ifdef _WIN32 check.